__tests__/pages/p/[id].test.jsx

```diff
@@ -6,19 +6,18 @@ import Page from '../../../pages/p/[id].tsx';
 import { getServerSideProps } from "../../../pages/p/[id].tsx"
```

Comment:
Use https://www.typescriptlang.org/tsconfig/#paths to create shortcuts for these paths.